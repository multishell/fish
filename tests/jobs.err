bg: '3' is not a job
fg: No suitable job: 3
