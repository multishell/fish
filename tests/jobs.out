1
Command
sleep
sleep
0
Command
sleep
sleep
